GRK_Project/Debug/GRK_Project.tlog/GRK_Project.lastbuildstate

```diff
@@ -1,2 +1,2 @@
-PlatformToolSet=v143:VCToolArchitecture=Native32Bit:VCToolsVersion=14.34.31933:TargetPlatformVersion=10.0.19041.0:
-Debug|Win32|D:\Coding\GRK - Project\|
+PlatformToolSet=v142:VCToolArchitecture=Native32Bit:VCToolsVersion=14.29.30133:TargetPlatformVersion=10.0.19041.0:
```

Comment:
na remote labsach sa tylko build toolsy v142

Reply:
co do blokowanie wyjscia z domu to wolalbym nie, chcialem dodac animacje otwierania drzwi kiedy podlecimy bliżej, żeby wejsc do domu